index.html

```diff
@@ -710,6 +716,16 @@ <h3>
             </p>
           </li>
         </ol>
+        <p data-dfn-for="PerformanceResourceTiming">
+          The <dfn>responseStatus</dfn> getter steps are to return
+          <a>this</a>'s <a data-for="PerformanceResourceTiming">response status</a>.
+        </p>
+        <p class='note'>
+          `responseStatus` is determined in [=Fetch=]. For a <a data-cite=
+          "FETCH#dom-requestmode-no-cors">no-cors</a> request it would be 0
```

Comment:
Is that also true for same origin no-cors responses?

Reply:
Its only 0 if it's cross origin no-cors....updated it. Thanks!!